config/application.rb

```diff
@@ -35,7 +35,7 @@ class Application < Rails::Application
     config.generators.assets = false
     config.generators.helper = false
 
-    config.active_job.queue_adapter = :sucker_punch
+    config.active_job.queue_adapter = :delayed_job
```

Comment:
Ahh :laughing:

Reply:
I'm sure we will regret this step. Any background job framework adds plenty of complexity. I had only bad developer experience in the past with either of these.